tools/scoring/score/dimensions/dimension.py

```diff
@@ -133,14 +141,17 @@ def __init__(self,
     self.correct_ceiling_override: int = None
     self.incorrect_total_override: int = None
 
-    if not translations and not deserialized_files:
+    translations_truthy_or_empty = translations or isinstance(
```

Comment:
what do you mean by truthy here?

Reply:
Truthy means it evaluates to `True`. In this case, allowing for it to be empty is necessary to allow for file sets which did not have any matching entities.